build-setup-nolog.sh

```diff
@@ -173,36 +173,10 @@ fi
 # When FireSim is being used as a library, the user is expected to build their
 # own toolchain. For FireSim-as-top, call out to Chipyard's toolchain scripts.
 if [ "$SKIP_TOOLCHAIN" != true ]; then
```

Comment:
Would we not want to create this environment whenever `IS_LIBRARY` is set false, regardless of what we are doing with the toolchain? 

Does it even make sense to have the skip-toolchain functionality if conda is bringing it in fairly cheapily?

Reply:
Agreed. The environment should always be created. However, I think we should still have the skip toolchain option for power users who need their own toolchain

Reply:
Because I think that Chipyard is going to want to create a conda environment when they are "top" and it seemed to make sense that firesim would cede it's ownership of the environment when it isn't "top"

Reply:
To be more clear: FireSim as top implies IS_LIBRARY = false, it does _not_ imply SKIP_TOOLCHAIN = true currently. 

For instance, i often skip building the toolchain, like on mill where i can't use the prebuilt one, when using FireSim as top currently.